src/font/pango/escape.hpp

```diff
@@ -55,6 +55,9 @@ inline std::string semi_escape_text(const std::string & text)
 	for(const char c : text) {
 		if(c == '&') {
 			ss << "&amp;";
+		} else if(c == '<') {
+			/* escape < so selection works */
+			ss << "&lt;";
```

Comment:
Is this correct? Where is this function used? Is it okay to escape `<` in all those cases?

Reply:
Removed.

Reply:
No longer needed since I'm using pango background attribute instead of markup to draw the highlight.